apis/rds/v1alpha1/custom_types.go

```diff
@@ -106,3 +106,18 @@ type CustomDBClusterParameters struct {
 	// +optional
 	SkipFinalSnapshot bool `json:"skipFinalSnapshot,omitempty"`
 }
+
+// CustomGlobalClusterParameters are custom parameters for a GlobalCluster
+type CustomGlobalClusterParameters struct {
+	// SourceDBClusterIDRef is a reference to a DBCluster used to set
+	// SourceDBClusterIdentifier.
+	// +immutable
+	// +optional
+	SourceDBClusterIDRef *xpv1.Reference `json:"sourceDBClusterIDRef,omitempty"`
+
+	// SourceDBClusterIDSelector selects a reference to a DBCluster used to
+	// set SourceDBClusterIdentifier.
+	// +immutable
+	// +optional
+	SourceDBClusterIDSelector *xpv1.Selector `json:"sourceDBClusterIDSelector,omitempty"`
```

Comment:
```suggestion
	SourceDBClusterIdentifierSelector *xpv1.Reference `json:"sourceDBClusterIdentifierSelector,omitempty"`
```

We use the exact name of the field as prefix for consistencty.